README.md

```diff
@@ -1,30 +1,41 @@
 # Deep Security Smart Check Scan Action
 
-For scanning your images as part of your CI pipeline using [Deep Security Smart
-Check][].
+<img src="Trend-Micro-Logo.png">
 
-[deep security smart check]: https://www.trendmicro.com/smartcheck
+## Scan your containers with [Deep Security Smart Check](https://www.trendmicro.com/smartcheck).
 
-This tool is used by the Deep Security Smart Check plugin for Jenkins and can
-also be used as a [GitHub Action](https://github.com/features/actions).
+This tool is used by the Deep Security Smart Check plugin for [Jenkins](https://plugins.jenkins.io/deepsecurity-smartcheck/) and can also be used as a [GitHub Action](https://github.com/features/actions).
+
+## Requirements
+
+* Have an [Deep Security Smart Check](https://www.trendmicro.com/smartcheck) deployed. [Sign up for free trial now](https://www.trendmicro.com/product_trials/download/index/us/168) if it's not already the case!
+* A container image to scan in any [supported Docker Registry](https://deep-security.github.io/smartcheck-docs/admin_docs/admin.html#supported-registries).
```

Comment:
I was unable to get the Smart Check scan to work with Google Container Registry. It didn't seem to like the way I was specifying the credentials to access the GCR. If it's possible to get this action to work with GCR we should amend the documentation to demonstrate this. To be sure, Smartcheck is certainly capable of scanning a GCR registry, but I'm not sure how to trigger this via the API per the documentation.

Reply:
I'll write something to cover that 👍